marklogic-data-hub-central/ui/src/components/header/system-info.test.tsx

```diff
@@ -106,13 +118,64 @@ describe("Update data load settings component", () => {
     expect(getByText(`Are you sure you want to clear all user data? This action will reset your instance to a state similar to a newly created DHS instance with your project artifacts.`));
     let confirm = getByLabelText("Yes");
     fireEvent.click(confirm);
-    expect(axiosMock.post).toBeCalledWith("/api/environment/clearUserData");
+    expect(axiosMock.post).toBeCalledWith("/api/environment/clearUserData", {});
 
     expect(await(waitForElement(() => getByText((content, node) => {
       return getSubElements(content, node, "Clear All User Data completed successfully");
     })))).toBeInTheDocument();
   });
 
+  test("Verify project info display, user with \"Clear\" button enabled and options work correctly", async () => {
+    mocks.clearUserDataAPI(axiosMock);
+    const authorityService = new AuthoritiesService();
+    authorityService.setAuthorities(["clearUserData"]);
+
+    const {getByText, getByTestId, getByLabelText, getByPlaceholderText} = render(<Router><AuthoritiesContext.Provider value={authorityService}>
+      <SystemInfo
+        systemInfoVisible={true}
+        setSystemInfoVisible={jest.fn()}
+      />
+    </AuthoritiesContext.Provider></Router>);
+
+    await(() => expect(axiosMock.get).toHaveBeenCalledTimes(1));
+
+    await wait(()  => {
+      expect(getByText("Clear")).toBeEnabled();
+      expect(getByTestId("deleteAll")).toBeChecked();
+      expect(getByText("Select a Database:")).toBeInTheDocument();
+      expect(getByLabelText("targetDatabase-select")).not.toBeEnabled();
+      expect(getByLabelText("targetBasedOn-select")).not.toBeEnabled();
+
+      fireEvent.click(getByTestId("deleteSubset"));
+      expect(getByTestId("deleteSubset")).toBeChecked();
+
+      expect(getByLabelText("targetDatabase-select")).toBeEnabled();
+      expect(getByLabelText("targetBasedOn-select")).toBeEnabled();
+      expect(getByTestId("targetBasedOnOptions-None")).toBeInTheDocument();
+      expect(getByText("Clear")).toBeEnabled();
+
+      fireEvent.keyDown(getByLabelText("targetBasedOn-select"), {key: "ArrowDown"});
+      expect(getByText("Collection")).toBeInTheDocument();
+      fireEvent.click(getByText("Collection"));
+      const collectionsInput = getByPlaceholderText("Search collections");
+      expect(collectionsInput).toBeInTheDocument();
+
+      expect(getByText("Clear")).not.toBeEnabled();
+      fireEvent.keyDown(getByLabelText("targetBasedOn-select"), {key: "ArrowDown"});
+      expect(getByText("Entity")).toBeInTheDocument();
+      fireEvent.click(getByText("Entity"));
+      const entitiesInput = getByPlaceholderText("Search entities");
+      expect(entitiesInput).toBeInTheDocument();
+      expect(getByText("Clear")).not.toBeEnabled();
+
+      userEvent.type(entitiesInput, "cust");
+      expect(getByLabelText("Customer")).toBeInTheDocument();
+      fireEvent.click(getByLabelText("Customer"));
+      expect(getByText("Clear")).toBeEnabled();
+
```

Comment:
The RTL test looks good and so close to perfect! Is it possible to add one last check of clicking the "Clear" button and verifying the endpoint is called with the collection and database in the payload? Something like:
`    expect(axiosMock.post).toBeCalledWith("/api/environment/clearUserData", payload);
`
where 
`payload = { targetCollection: "Customer", targetDatabase: "STAGING"}`

Reply:
Added. I had some issue when checking the modal text. But left a comment to include it once the updated text story is worked on.